src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/resource/VisitResource.kt

```diff
@@ -71,14 +72,27 @@ class VisitResource(private val visitService: VisitService) {
       ),
     ]
   )
-  fun createVisit(@RequestBody @Valid createVisitRequest: CreateVisitRequest): CreateVisitResponse =
-    visitService.createVisit(createVisitRequest)
+  fun createVisit(
+    @Schema(description = "Offender Noms Id", example = "A1234ZZ", required = true)
+    @PathVariable
+    @Pattern(regexp = OFFENDER_NO_PATTERN)
+    offenderNo: String,
+    @RequestBody @Valid createVisitRequest: CreateVisitRequest
+  ): CreateVisitResponse =
+    visitService.createVisit(offenderNo, createVisitRequest)
```

Comment:
Not sure about whether OFFENDER_NO_PATTERN is worth it, i was thinking about catching dodgy chars in the url

Reply:
yeah, I'm never sure about that either, given an invalid offenderNo would be caught at some point since the offender wouldn't exist